webapp/models.py

```diff
@@ -57,7 +57,7 @@ class CVE(db.Model):
     updated_at = Column(
         DateTime(timezone=True), server_default=func.now(), onupdate=func.now()
     )
-    notes = Column(JSON)
+    notes = Column(JSON)  # JSON array
```

Comment:
Important comment. Do not initialize test data as a JSON object or anything else.